next/components/pages/DocumentPage.tsx

```diff
@@ -87,7 +87,7 @@ const DocumentPage = ({ entity }: IProps) => {
                 >
                   {/* Change to 'Open' when download button is added */}
                   {/* {t('open')} */}
-                  {t('download')}
```

Comment:
Just to mention that if the file is not "openable in browser" (such as pdf is), it is actually downloaded.
I thinks it is also common practice to open the file in new tab before download.
But I assume this comes from the client, I just wanted to mention it.